internal/app/http/jwt/jwks.go

```diff
@@ -0,0 +1,49 @@
+package jwt
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/MicahParks/keyfunc/v3"
+)
+
+// TODO(pedram): set this to 15 Min after testing
+const MinIntervalDuration = 2 * time.Second
+
+type JWTValidationOpts struct {
+	JWTServiceID  string
+	JWKSURLString string
+	JWKSInterval  time.Duration
+}
+
+func validateOptions(opts JWTValidationOpts) {
+	// disallow intervals that are too short
+	if opts.JWKSInterval < MinIntervalDuration {
```

Comment:
This currently prevents starting the server, as JWKSInternal is not set anywhere.  
What is the purpose of the JWKSInternal, as it is only used at the moment in this line ?